x-pack/plugins/observability/public/pages/slos/components/grouped_slos/group_view.tsx

```diff
@@ -24,10 +24,9 @@ interface Props {
 }
 
 export function GroupView({ isCompact, kqlQuery, sloView, sort, direction, groupBy }: Props) {
-  const [page, setPage] = useState(0);
   const [perPage, setPerPage] = useState(DEFAULT_SLO_GROUPS_PAGE_SIZE);
-  const { state } = useUrlSearchState();
-  const { tagsFilter, statusFilter, filters } = state;
+  const { state, store: storeState } = useUrlSearchState();
+  const { tagsFilter, statusFilter, filters, page } = state;
```

Comment:
@dominiqueclarke Need to test this, but if I recall correctly I was not using the urlState for the page, because the page is referring to the outer pagination, where as I wanted to refer to the inner pagination within a group. What happens if you paginate within a group? Does everything work as expected? I will create a few a SLOs and will give it a try.